allan_variance.py

```diff
@@ -53,9 +53,15 @@ def allan_variance(x, dt=1, min_cluster_size=1, min_cluster_count='auto',
     tau : ndarray
         Averaging times for which Allan variance was computed, 1-d array.
     avar : ndarray
-        Values of AV. The 0-th dimension is the same as for `tau`. The trailing
-        dimensions match ones for `x`.
+        Values of AVAR. The 0-th dimension is the same as for `tau`. The
+        trailing dimensions match ones for `x`.
+
+    References
+    ----------
+    .. [1] https://en.wikipedia.org/wiki/Allan_variance
     """
+    if input_mode.lower() not in ("increment", "mean"):
```

Comment:
For history. I looked at the scipy code. For example, [here](https://github.com/scipy/scipy/blob/maintenance/0.18.x/scipy/sparse/csgraph/_traversal.pyx#L64):
```python
if connection.lower() not in ['weak', 'strong']:
    raise ValueError("connection must be 'weak' or 'strong'")
```